sw/airborne/modules/computer_vision/viewvideo.c

```diff
@@ -100,6 +102,11 @@ void *computervision_thread_main(void *data)
   vid.device = (char *)"/dev/video1";
   vid.w = 1280;
   vid.h = 720;
+//#if BOTTOM_CAMERA == 1
```

Comment:
check is commented out?
Also would probably be nicer `#if USE_BOTTOM_CAMERA` or `#if VIEW_BOTTOM_CAMERA`